Docker-compose/docker-compose.yml

```diff
@@ -9,8 +9,7 @@ services:
       MONGO_INITDB_ROOT_USERNAME: "${MONGO_DBUSER:-satosa}"
       MONGO_INITDB_ROOT_PASSWORD: "${MONGO_DBPASSWORD:-thatpassword}"
     volumes:
-      - mongodata:/data/db
-      - /usr/share/zoneinfo/Europe/Rome:/etc/localtime:ro
```

Comment:
This Is needed for the local timezone within the container

Reply:
I don't think it's necessary, this work is already execute in line 23 of Dockerfile
https://github.com/italia/Satosa-Saml2Spid/blob/b26f78fb330e8c09d000cc569f713c9201ab9700/Dockerfile#L23

Reply:
Hi @peppelinux , Apologize, I didn't realize you were referring to MongoDB.

MongoDB save the datetime object in UTC format (without explicit rule), for this container  the timezone is not very important. Anyway, MongoDB official image, support the TZ environment  to set the timezone:

```bash
[ranaldi@andrea-ranaldi-p ~]$ docker exec -tie TZ=Europe/Rome mongo bash
root@b8714ec55ff8:/# date
Mon Aug 26 15:59:00 CEST 2024
root@b8714ec55ff8:/# exit
exit
[ranaldi@andrea-ranaldi-p ~]$ docker exec -tie TZ=Australia/Melbourne mongo bash
root@b8714ec55ff8:/# date
Mon Aug 26 23:59:13 AEST 2024
root@b8714ec55ff8:/# 
```
I add the TZ environment in the docker-compose file

Reply:
I don't want that the docker image uses a static timezone, but it would be better to have it configurable through local mount points

for all the rest in this PR I fully approve the approach that really simplify all the things!

thx @MdreW